ui/admin/app/components/form/auth-method/oidc/index.js

```diff
@@ -27,4 +30,48 @@ export default class FormAuthMethodOidcComponent extends Component {
    * @type {string}
    */
   newSigningAlgorithm = options.oidc.signing_algorithms[0];
+
+  /**
+   * @type {string}
+   */
+  prompts = options.oidc.prompts;
+
+  @tracked selectedPrompts = this.parsePromptsArray();
+
+  /**
+   * @returns {string}
+   */
+  parsePromptsArray() {
+    return (this.args.model.prompts || []).map((item) => item.value);
+  }
+
+  //actions
+
+  /**
+   * @param {string} option
+   * @param {object} event
+   */
+  @action
+  updatePrompt(option, event) {
+    const currentSelection = option.split(',').map((value) => ({ value }));
+
+    const previousSelection = this.args.model.prompts || [];
+
+    //when the checkbox is unchecked, remove the option form the exisiting list
+    const removeSelection = previousSelection.filter((i) => i.value !== option);
```

Comment:
None blocking comment: totally personal opinion, and I recognize a picky comment, that's why I mention is none blocking and feel free to ignore it if you disagree 😉 But I would like the var `i` is a bit more descriptive, perhaps `item`?